### ISSUE TITLE
Testing the module

### ISSUE THREAD

**Issue by NetanelBasal** [moff/angular2-module-starter#1]
What about testing the module?

**Comment by moff** [comment]
Hi @NetanelBasal!

This repo doesn't have tests currently and there is not much to test since it's just a module starter. But I think that's a good idea to provide some basic starting point for testing inside the starter for people who will develop modules on top of it.

I'll add this feature as soon as possible.

Thanks for a feedback!

**Comment by NetanelBasal** [comment]
Yes that's what i meant, testing for the module not the repo. Thanks.
